pkg/controller/appdefinition/pullappimage.go

```diff
@@ -13,12 +13,15 @@ import (
 	"github.com/acorn-io/acorn/pkg/images"
 	"github.com/acorn-io/acorn/pkg/tags"
 	"github.com/acorn-io/baaah/pkg/router"
+	imagename "github.com/google/go-containerregistry/pkg/name"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const defaultNoReg = "xxx-no-reg"
```

Comment:
There is an exported `const` in the `pkg/images` package called `DefaultRegistry`. Can this exported constant be changed (I think `xxx-no-reg` makes more sense) and be used everywhere we need it?